docs/UserGuide.md

```diff
@@ -313,6 +316,7 @@ All commands in this section are subjected to the restriction in this table exce
 `edit_c 3 name=Ryan Koh`
 
 * Edit the name of the 3rd candidate in the list to Ryan Koh.
+* If a candidate has an [interview](#feature-interview-management), edit a status that is not `Applied` back to `Applied` is not allowed, and will be automatically changed to `Scheduled`. You can choose the change to other status such as `None`.
```

Comment:
Typo: edit to editing.

You can _choose to change_ to other.....

Reply:
Made the changes accordingly.